app/models/job.rb

```diff
@@ -1,4 +1,28 @@
 class Job < ActiveRecord::Base
-  validates :title, :level_of_interest, :city, presence: true
+  validates :title, :level_of_interest, :city, :category_id, presence: true
+
   belongs_to :company
+  belongs_to :category
+  has_many :comments, dependent: :destroy
+
+  def self.location
```

Comment:
i don't see tests for these methods either. don't forget to test every method you write in your models!